src/log.c

```diff
@@ -140,16 +141,23 @@ char *ices_log_get_error(void) {
 
 /* Function to log string to both console and file */
 static void ices_log_string(char *format, char *string) {
+	time_t t = time(NULL);
+	char tstr[48];
+	strftime(tstr, sizeof(tstr), "%Y.%m.%d %H:%M:%S", localtime(&t));
```

Comment:
Is it possible to use hyphens in timestamps and use a different timestamp separator (no 4th colon)? The final result could be like this:
```
2020-09-07 20:21:22 - Playing /path/to/track1
2020-09-07 20:23:22 - Playing /path/to/track2
```

At the moment it's more like this:
```
2020.09.07 20:21:22: Playing /path/to/track1
2020.09.07 20:23:22: Playing /path/to/track2
```